### PR TITLE
Zhourao Initial work

### PR DESCRIPTION
**Pull request by Xerodoeht**
The gamerule is functional but there are no events to give out states yet. The spline work and all that is done however.

### CODE REVIEW

File: common/technology/technologies/dvg_techs.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿
+
+dvgaustralia = {
+	# Unlocks construction of Farms and Plantations
```
Comment (MadJamez) [inline common/technology/technologies/dvg_techs.txt:4]: This comment is wrong :D

File: localization/english/dvg_flavor_oceania_l_english.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+﻿l_english:
+ #State Traits
+ state_trait_dvg_australia_locked: "Australia Gamerule"
```
Comment (MadJamez) [inline localization/english/dvg_flavor_oceania_l_english.yml:3]: Maybe just Australian Colonization or something?
In Game rules they can define the gamerule.

File: common/ai_strategies/00_default_strategy.txt [hunk 3]
```diff
@@ -809,6 +809,18 @@ ai_strategy_default = {
 			add = 100
 		}
 		
+		#Please stop killing your vassal burgundy
+		if = {
```
Comment (MadJamez) [inline common/ai_strategies/00_default_strategy.txt:813]: Maybe it is better to add something in secret goal to make them more likely to be protective?

**Review by MadJamez — CHANGES_REQUESTED** [review]

File: common/history/states/dvg_oceania.txt [hunk 4]
```diff
@@ -2,55 +2,74 @@
...
 		}
 		
 		add_homeland = zhourao
 		add_homeland = aborigine
 	}
-	s:STATE_PARLEVAR = {
+	
+	#Zhushan
+	s:STATE_WENCHENG = {
 		create_state = {
```
[the remark this answers is not in the document]
Comment (Xerodoeht) [reply common/history/states/dvg_oceania.txt:24]: I have fixed this three times, it keeps unfixing itself, wtf

[the remark this answers is not in the document]
Comment (Xerodoeht) [reply common/history/states/dvg_oceania.txt:24]: The moment I fixed it, xA111C7 is now crossing multiple state regions

File: common/ai_strategies/00_default_strategy.txt (hunk 3 above)
Comment (Xerodoeht) [reply common/ai_strategies/00_default_strategy.txt:813]: Will do in future, rn the event for them to exist doesn't trigger yet

**Review by MadJamez — APPROVED** [review]

File: localization/english/dvg_flavor_oceania_l_english.yml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+ #State Traits
+ state_trait_dvg_australia_locked: "Australian Colonization"
+ state_trait_murray_irrigation: "Great Yongding"
+ 
+ #Tech
+ dvgaustralia: "Australia Colonizable"
+ dvgaustralia_desc: "This technology allows all nations to colonize Australia, it is set by an option in gamerules. It will most likely result in terrifying bordergore."
+
+ #Events
+ dvg_oceania.1.t: "New Zealand"
```
Comment (Glloyd) [inline localization/english/dvg_flavor_oceania_l_english.yml:11]: Should this be Nouvelle Zélande?

File: localization/english/dvg_flavor_zhourao_l_english.yml [hunk 6]
```diff
@@ -0,0 +1,38 @@
+﻿l_english:
+ #Events
+ dvg_oceania.1.t: "New Zealand"
```
Comment (Glloyd) [inline localization/english/dvg_flavor_zhourao_l_english.yml:3]: likewise here

**Review by Glloyd — CHANGES_REQUESTED** [review]
looks good aside from the above. I'm assuming its intentional that Mianjin is using a generic flag right now?

**Comment by Glloyd** [comment]
beat me by 5 seconds lmao

**Pull request merged**